### DISCUSSION TITLE
gettin-nacky/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#54] [Announcements]
# gettin-nacky/

Rejection
It’s hard. Nobody likes that feeling, especially after putting in a bunch of work, double-especially when that work is on a Wayland protocol.

https://www.supergoodcode.com/gettin-nacky/

**Comment by Permik** [comment]
Mike, you really have a way with words.  
PACK:ing (Positive ACKnowledge) this NACK-proposal.

**Comment by Eclipse32767** [comment]
Sebastian Wick is absolutely sobbing right now how dare you /j
